### PR TITLE
Topic/update skia to m87

### PR DESCRIPTION
**Pull request by flsobral**
Integration step of #196

### CODE REVIEW

File: TotalCrossVM/docker/arm32v7/Dockerfile [hunk 1]
```diff
@@ -10,7 +9,6 @@ RUN apt-get update && \
         ninja-build \
         build-essential \
         libfontconfig1-dev \
-# SDL2
```
Comment (flsobral) [inline TotalCrossVM/docker/arm32v7/Dockerfile:13]: is there a reason to remove this? I like it.

File: TotalCrossVM/docker/arm32v7/Dockerfile [hunk 2]
```diff
@@ -33,13 +32,12 @@ RUN apt-get update && \
         fcitx-libs-dev \
         libsamplerate0-dev \
         libsndio-dev \
-# Wayland
```
Comment (flsobral) [inline TotalCrossVM/docker/arm32v7/Dockerfile:36]: is there a reason to remove this? I like it.

File: TotalCrossVM/docker/arm32v7/Dockerfile [hunk 3]
```diff
@@ -33,13 +32,12 @@ RUN apt-get update && \
...
         libwayland-dev \
         libxkbcommon-dev \
         wayland-protocols \
         git \
         ca-certificates; \
-    apt-get install -y libgles1-mesa-dev || apt-get -f install; \
+    apt-get install -y libglvnd-dev || apt-get -f install; \
```
Comment (flsobral) [inline TotalCrossVM/docker/arm32v7/Dockerfile:40]: why has this been changed?

File: TotalCrossVM/docker/arm32v7/Dockerfile [hunk 4]
```diff
@@ -53,24 +51,9 @@ RUN git clone https://github.com/SDL-mirror/SDL.git \
...
-# utilize o diretorio TotalCrossVM como diretorio de contexto de build para o Docker
-COPY . /sources
-
-RUN cmake /sources -G Ninja && ninja
-
-CMD ["/bin/bash", "-c", "make", "-j$(($(nproc) + 2))", "-f", "${BUILD_FOLDER}/Makefile"]
-# CMD ["make", "clean"]
-
-FROM scratch AS export
-COPY --from=build /build /
```
Comment (flsobral) [inline TotalCrossVM/docker/arm32v7/Dockerfile:76]: won't this change break the ci/cd?

File: TotalCrossVM/docker/arm32v7/build-docker.sh [hunk 5]
```diff
@@ -4,4 +4,4 @@ export DOCKER_BUILDKIT=1
 docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 docker buildx create --use --name armbuilder
 docker buildx inspect --bootstrap
-docker buildx build --platform linux/arm/v7 --load -t totalcross/linux-arm32v7-build .
+docker buildx build --no-cache --platform linux/arm/v7 --load -t totalcross/linux-bionic-arm32v7-build .
```
Comment (flsobral) [inline TotalCrossVM/docker/arm32v7/build-docker.sh:7]:
```suggestion
docker buildx build --platform linux/arm/v7 --load -t totalcross/linux-bionic-arm32v7-build .
```
commited the --no-cache by mistake

**Review by flsobral — COMMENTED** [review]
squash the commits that updates the binaries into a single one

File: TotalCrossVM/docker/arm32v7/Dockerfile (hunk 3 above)
Comment (Farshid-Monhaseri) [reply TotalCrossVM/docker/arm32v7/Dockerfile:40]: As I remember Ubuntu bionic was compiling about package 'libgles1-mesa-dev' and It recommends to use 'libglvnd-dev' as replacement package

**Review by Farshid-Monhaseri — COMMENTED** [review]
The size of arm64 build is not logical ... The size of the arm32&64 builds are not coherent ... They should be close, unless the build options for them are considerably different.

File: TotalCrossVM/docker/arm32v7/Dockerfile (hunk 3 above)
Comment (flsobral) [reply TotalCrossVM/docker/arm32v7/Dockerfile:40]: @acmlira the commit is missing this information

**Review by acmlira — APPROVED** [review]

File: TotalCrossVM/CMakeLists.txt [hunk 6]
```diff
@@ -525,12 +525,12 @@ target_compile_options(tcvm
 # We need to use C++ 11
 if(${CMAKE_VERSION} VERSION_LESS "3.8" OR MSVC) 
   set_target_properties(tcvm PROPERTIES
-    CXX_STANDARD 11
```
[the remark this answers is not in the document]
Comment (flsobral) [reply TotalCrossVM/CMakeLists.txt]: Open an issue about reviewing our CMake base support please.

**Pull request merged**